app/assets/javascripts/application/evaluationtable.js

```diff
@@ -86,6 +86,25 @@ var evaluationTable = (function($) {
     localStorage.setItem('hiddenUsers', JSON.stringify(list));
   }
 
+  module.refreshSumOfPrinciple = function(input) {
+    var principle_id = input.getAttribute('data-principle');
+
+    total = calcSum(principle_id);
+
+    $('#sum-of-principle-' + principle_id).html(total);
+  }
+
+  function calcSum(principle_id) {
+    var total = 0;
+
+    principleArray = $('*[data-principle="' + principle_id + '"]');
+    for (var i = 0; i < principleArray.length; i++){
```

Comment:
Its maybe a bit cleaner to do the summing in a [reduce function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) rather than a for loop with an additional variable.

Although you will need to convert the jQuery object to an array with `toArray()`, because they are not real arrays, they just behave like it.